styles.css

```diff
@@ -9,37 +9,78 @@
 body {
   font-family: 'lato', sans-serif;
   font-size: small;
+  line-height: 1.5;
 }
 
 button {
   background: none;
   text-align: center;
   padding: 0.125em 0.5em;
   border-radius: 3px;
-  box-shadow: 1px 1px;
+  border: 2px solid rgba(0, 0, 0, 0.5);
+  box-shadow: 1.5px 1.5px;
   cursor: pointer;
 }
 
 ul {
   list-style: none;
 }
 
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.hidden {
+  display: none;
+}
+
 .container {
   width: 70%;
   max-width: 700px;
   margin: 0 auto;
+  min-height: 100vh;
+  padding: 1.5em 0;
+}
```

Comment:
- [ ] Good job so far, However, Please ensure that you use the wireframe to guide you thoroughly, I can see that the width of your website does not take fullscreen and your date time does not match the design. Please resolve it. Following the given design helps meet user expectations. If users are already familiar with the design or have interacted with similar interfaces, your project's adherence to the provided design will make it easier for them to understand and use. Consistency with established design patterns and conventions reduces the learning curve and enhances usability. :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Accepted type</td>
    </tr>
    <tr>
      <td>

![Student Own](https://github.com/Xghayor/Awesome-Books/assets/43528594/ba4afb39-9288-4999-ab7e-189d0781bafd)

</td>
      <td>

![design guide](https://github.com/Xghayor/Awesome-Books/assets/43528594/67aa8009-31e2-46aa-be0c-21bb05422374)

</td>
    </tr>
  </table>